packages/jest-editor-support/src/__tests__/parsers/parserTests.js

```diff
@@ -19,6 +19,10 @@ function parserTests(parse: (file: string) => ParserReturn) {
     it('For the simplest it cases', () => {
       const data = parse(`${fixtures}/global_its.example`);
 
+      if (data.skip) {
```

Comment:
Right, this means I can just change the package.json and revert everything else.
Should I also add a note in the README explaining that if someone wants to use `.tsx` they need to provide TypeScript and if they are not interested into TS support they can safely ignore the warning?